source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,310 @@
+Bulk export data
+=================
+At this time, the export supports attributes of the objects listed below.   Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. Deleted objects are also not yet supported.  For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea formum <https://mattermost.uservoice.com/forums/306457-general>`_.  
```

Comment:
`formum` --> `forum`

Reply:
Propose having `For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea formum <https://mattermost.uservoice.com/forums/306457-general>`_.  ` on a separate paragraph to separate call for contributions.